packages/view/src/Charcoal/View/Twig/TwigEngine.php

```diff
@@ -30,10 +40,113 @@ public function type(): string
         return 'twig';
     }
 
+    /**
+     * Build the Mustache Engine with an array of dependencies.
+     *
+     * @param array $data Engine dependencie.
+     */
+    public function __construct(array $data)
+    {
+        parent::__construct($data);
+
+        $this->config = $data['config'];
+        $this->debug = $data['debug'];
+
+        if (isset($data['helpers'])) {
+            $this->setHelpers($data['helpers']);
+        }
+    }
+
+    /**
+     * Set the engine's helpers.
+     *
+     * @param  array|Traversable|HelpersInterface $helpers Mustache helpers.
+     * @throws InvalidArgumentException If the given helper(s) are invalid.
+     * @return MustacheEngine Chainable
+     */
+    public function setHelpers($helpers)
+    {
+        if ($helpers instanceof HelpersInterface) {
+            $helpers = $helpers->toArray();
+        }
+
+        if (!is_array($helpers) && !$helpers instanceof Traversable) {
```

Comment:
Either use `iterable` pseudo-type (introduced in PHP 7.1) or `is_iterable()` function.

```suggestion
        if (!is_iterable($helpers)) {
```